traffic_portal/app/src/common/modules/table/jobs/TableJobsController.js

```diff
@@ -80,8 +80,6 @@ var TableJobsController = function(tableName, jobs, $document, $scope, $state, $
 			x.expires = new Date(x.startTime.getTime() + ttl*3600*1000);
 		});
 
-	$scope.jobs = jobs;
```

Comment:
why?

Reply:
yeah, this is unrelated to this PR but i didn't want to create a whole new PR for it. this shadow variable seems to do no harm to the jobs table but it should be removed imo.